test/common/test_query_children.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2015 Cisco Systems, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nose.tools import *
+from ..connection.info import custom_setup, custom_teardown
+
+handle = None
+user_list = []
+
+
+def setup_module():
+    from imcsdk.mometa.aaa.AaaUser import AaaUser
+
+    global handle
+    handle = custom_setup()
+    # user_class = handle.query_dn("sys/user-ext")

```

Comment:
@prvashis should the code below be uncommented? This is a test for query_children and the commented part is what creates those children I assume.